### ISSUE TITLE
possible automatic filtering of links

### ISSUE THREAD

**Issue by laurajohman** [GrafeasGroup/tor_ocr#17]
this might not work in practise. 

Currently (10th March 18) the bot will transcribe links from bit (dot) ly, goo . gl and other link shortening companies from images it receives. However this triggers the the post to be automatically filtered through the modqueue. I usually fish them out of the modqueue when I get a notifcation - however it might be easier if there was a thing that allowed the link text to be replaced with a [redacted link]. 

there's probably issues with that (when the bot hits a link from then on there's just [redacted link] and no further transcription done, crashing, new link shortening companies around) but it's just something to think about.

**Comment by TheLonelyGhost** [comment]
I'm a bit unclear still on what is being suggested. Let me see if I can explain it back to you and you fill in the gaps&mdash;or possibly I'm completely off in the weeds and spouting off nonsense.

## Current behavior

OCR bot attempts to transcribe links that are provided via url shorteners (e.g., https://bit.ly/, https://goo.gl/, etc.). Assuming those shortened links resolve to an image, it attempts to transcribe them. The transcription comment the OCR bot leaves contains content which triggers a spam filter, requiring us to manually fish it out of the mod queue.

## Suggested behavior

When the OCR bot posts the comment, it should replace the shortened url (e.g., [goo.gl/vJMxrv](https://goo.gl/vJMxrv)) with "[redacted link]". This would prevent the spam filter from catching it.

Possible obstacles might be keeping up with all of the possible url shortening services.

---

@laurajohman Is that pretty close to what you were suggesting?

**Comment by laurajohman** [comment]
Yes. The bot "reads" the image and outputs the text. However this leaves 'live' links in the text output comment on /r/TOR. Because the links are live, the comment gets sent to the spam filter. 

I was proposing a self censor - if the bot reads a bit.ly link, or a link shorter link
